app/views/api/v1/users/_user.json.jbuilder

```diff
@@ -1,5 +1,12 @@
-json.id user.id
-json.first_name user.first_name
-json.last_name user.last_name
-json.username user.username
-json.email user.email
\ No newline at end of file
+if user == current_user
+    json.id user.id
+    json.first_name user.first_name
+    json.last_name user.last_name
+    json.username user.username
+    json.email user.email
+else
+    json.id user.id
+    json.first_name user.first_name
+    json.last_name user.last_name
+    json.events user.events
```

Comment:
I think this needs to render a collection/partial or whatever

Reply:
wait what do you mean? isn't this a partial?